alibi/explainers/cfproto.py

```diff
@@ -178,7 +178,11 @@ def __init__(self,
         self.max_iterations = max_iterations
         self.c_init = c_init
         self.c_steps = c_steps
-        self.feature_range = feature_range
+        self.feature_range = list(feature_range)
```

Comment:
Nitpick: might be slightly cleaner to just `feature_range = list(feature_range)` and assign as an attribute once we're done with the pre-processing.